corehq/apps/case_importer/do_import.py

```diff
@@ -332,6 +354,15 @@ def _get_parent_index(self):
                     return {self.parent_ref: (parent_case.type, parent_case.case_id)}
                 raise exceptions.InvalidParentId(column)
 
+    def _get_host_index(self):
+        if self.host_id:
+            host_case, error = lookup_case('case_id', self.host_id, self.domain, self.host_type)
+            _log_case_lookup(self.domain)
```

Comment:
You didn't create this pattern, but I wonder if `_log_case_lookup` should just go inside of `lookup_case`. Although it'd need to have a param or something, since `lookup_case` is also used in repeaters and those don't log, so maybe it wouldn't be any simpler.

Reply:
Yeah, it's being used in other places. I had to remove this method as well, so it's simpler now